docs/development/plugins/data/public/kibana-plugin-plugins-data-public.search.md

```diff
@@ -27,8 +27,9 @@ search: {
         InvalidEsCalendarIntervalError: typeof InvalidEsCalendarIntervalError;
         InvalidEsIntervalFormatError: typeof InvalidEsIntervalFormatError;
         isDateHistogramBucketAggConfig: typeof isDateHistogramBucketAggConfig;
+        isNumberType: (agg: import("./search").AggConfig) => boolean;
         isStringType: (agg: import("./search").AggConfig) => boolean;
-        isType: (type: string) => (agg: import("./search").AggConfig) => boolean;
+        isType: (...types: string[]) => (agg: import("./search").AggConfig) => boolean;
```

Comment:
Where is this change coming from?

Reply:
Seems all of the docs changes were not directly related to changes in this PR,
because after the last merge with the master, all of them are just gone (means the same changes have been already merged -> https://github.com/elastic/kibana/blob/master/docs/development/plugins/data/public/kibana-plugin-plugins-data-public.search.md#search-variable)

Reply:
Weird, I thought we had a check for that in the build.

Reply:
CI checks that the latest `api.json` file is up to date and committed but doesn't verify that docs (based on this file) are up to date. so if you `--accept` api changes but don't commit the new docs this could happen.